mkdocs.yml

```diff
@@ -73,89 +62,98 @@ nav:
+          - Comparing PoA Consensus Protocols: Consensus-Protocols/Comparing-PoA.md
+        - Privacy:
+          - Overview: Privacy/Explanation/Privacy-Overview.md
           - Privacy Groups: Privacy/Explanation/Privacy-Groups.md
           - Processing Private Transactions: Privacy/Explanation/Private-Transaction-Processing.md
-    - Permissioning:
-        - Overview: Permissions/Permissioning-Overview.md
-        - Local Permissioning: Permissions/Local-Permissioning.md
-        - Onchain Permissioning:
-            - Overview: Permissions/Onchain-Permissioning/Onchain-Permissioning.md
-            - Getting Started with Onchain Permissioning: Permissions/Onchain-Permissioning/Getting-Started-Onchain-Permissioning.md
-            - Deploying Management Dapp for Production: Permissions/Onchain-Permissioning/Production.md
-            - Updating Whitelists: Permissions/Onchain-Permissioning/Updating-Whitelists.md
-    - Using Pantheon:
+        - Permissioning:
+          - Overview: Permissions/Permissioning-Overview.md
+          - Onchain Permissioning: Permissions/Onchain-Permissioning/Onchain-Permissioning.md
         - Transactions:
-          - Creating and Sending Transactions: Using-Pantheon/Transactions/Transactions.md
           - Transaction Pool: Using-Pantheon/Transactions/Transaction-Pool.md
           - Vadidating Transactions: Using-Pantheon/Transactions/Transaction-Validation.md
-          - Including Revert Reason: Using-Pantheon/Transactions/Revert-Reason.md
-        - Using Truffle with Pantheon: Using-Pantheon/Truffle.md
-        - Events and Logs:
-          - Overview: Using-Pantheon/Events-and-Logs.md
-          - Accessing Logs Using JSON-RPC API: Using-Pantheon/Accessing-Logs-Using-JSON-RPC.md
-        - Using Wallets for Account Management: Using-Pantheon/Account-Management.md
-        - Mining: Using-Pantheon/Mining.md
-    - Monitoring:
-        - Logging: Monitoring/Logging.md
-        - Monitoring Performance: Monitoring/Monitoring-Performance.md
-        - Monitoring Network:
-          - Alethio Overview: Monitoring/Alethio/Overview.md
-          - Alethio Ethereum Lite Explorer: Monitoring/Alethio/Lite-Block-Explorer.md
-          - Alethio EthStats Lite Network Monitor: Monitoring/Alethio/Lite-Network-Monitor.md
-    - Deploying Pantheon:
-        - Ansible: Deploying-Pantheon/Ansible.md
-        - High Availability of APIs: Deploying-Pantheon/High-Availability.md
-        - Migrating Docker Image: Deploying-Pantheon/Migration-Docker.md
-    - Troubleshooting:
-        - Troubleshooting: Troubleshooting/Troubleshooting.md
+        - Events and Logs: Using-Pantheon/Events-and-Logs.md
+        - Network vs Node Configuration: Configuring-Pantheon/Network-vs-Node.md
+        - Network ID and Chain ID: Configuring-Pantheon/NetworkID-And-ChainID.md
+        - Node Keys: Configuring-Pantheon/Node-Keys.md
+        - Alethio: Monitoring/Alethio/Overview.md
```

Comment:
Alethio doesn't fit well IMO in explanation. It should either go somewhere where we explain monitoring or in a 3rd party tools part.

Reply:
The other monitoring topics are in the How-To and this is conceptual information so doesn't belong there.  I'm going to raise a separate JI to look at this and add double-linking.  I like the idea of double-linking but need to think a bit more about it.

Reply:
https://pegasys1.atlassian.net/browse/PAN-3095